src/uint/boxed/sqrt.rs

```diff
@@ -0,0 +1,295 @@
+//! [`BoxedUint`] square root operations.
```

Comment:
This is fine to get this implemented, but in a followup PR (which I'm happy to do) it should probably get a similar macro treatment to the multiply functions so `Uint` and `BoxedUint` can share a common implementation.